docs/decisions/0003-token-exam-access.rst

```diff
@@ -0,0 +1,50 @@
+Token System for Exam Access
+=================================
+
+Status
+------
+
+Approved (circa September 2022)
+
+Context
+-------
+Exam integrity and security is a fundamental element of special exams on the edX platform, which includes
+timed exams, practice proctored exams, onboarding exams, and proctored exams. In order to ensure exam
+integrity and security, edX learners should only be able to see special exam content when they are authorized to do so.
+
+In the legacy experience for exams, ``edx-proctoring``, exam access was gated by frontend interstitials, as well as a
+check on the backend before rendering the student view. The access check on the backend was possible because the LMS
+directly imports ``edx-proctoring``, and has access to specific functions that return whether or not the content
+should be visible to a learner in an exam. More details on the legacy experience can be found in the `original discovery
+doc <https://2u-internal.atlassian.net/wiki/spaces/PT/pages/15440845/MST-1210+How+do+we+do+access+control+for+exam+content+without+edx-proctoring>`_.
+
+In the new experience, which uses the ``edx-exams`` IDA as a backend, the LMS cannot directly import any functionality from
+``edx-exams``, as it is an idea. Therefore, while the frontend gating still exists via the same interstitials from the legacy
```

Comment:
[Suggestion] this should be IDA right, not idea? Just want to make sure I'm reading correctly.

Reply:
Yes, must have autocorrected 👀